src/pages/partners/affiliate-ib/_deriv-affiliate-programme.tsx

```diff
@@ -206,152 +122,28 @@ const DerivAffiliateProgramme = () => {
-                                            mt_mobile="24px"
-                                        >
-                                            <Localize translate_text="_t_Sign up_t_" />
-                                        </ApplyNow>
-                                    </div>
-                                </StyledCard>
+                                <RevenueShareCard />
+                                <TurnoverCard />
                             </>
                         )}
-                        <StyledCard height="69rem" tabletHeight="auto" padding="2.4rem">
-                            <div>
-                                <Header as="h4" type="sub-section-title" mb="0.8rem">
-                                    <Localize translate_text="_t_CPA (EU only)_t_" />
-                                </Header>
-                                <Text>
-                                    <Localize translate_text="_t_Earn based on each successful referral._t_" />
-                                </Text>
-                                <CPAContent>
-                                    <Text>
-                                        <Localize
-                                            translate_text="_t_You earn <0>USD 100</0> when a referred client successfully deposits a one-time or cumulative total of <0>USD 100</0> into their Deriv account._t_"
-                                            components={[<strong key={0} />]}
-                                        />
-                                    </Text>
-                                    <Text>
-                                        <Localize
-                                            translate_text="_t_This plan is available exclusively for EU-based clients. <0>Please note that according to regulations, you cannot have clients who reside in Portugal or Spain.</0>_t_"
-                                            components={[<strong key={0} />]}
-                                        />
-                                    </Text>
-                                </CPAContent>
-                            </div>
-                            <div>
-                                <ApplyNow
-                                    id="dm-card-affiliate-signup-3"
-                                    secondary
-                                    to={affiliate_signup_url}
-                                    external
-                                    target="_blank"
-                                    type="affiliate_sign_up"
-                                    mt_mobile="32px"
-                                >
-                                    <Localize translate_text="_t_Sign up_t_" />
-                                </ApplyNow>
-                            </div>
-                        </StyledCard>
-                    </CardWrapper>
+                        <CpaEuOnlyCard />
+                        {!is_eu && <MasterAffilateCard />}
```

Comment:
can u check this on refresh is it working with out and glitch? on eu just refresh and check 
 ```suggestion
                        {(!is_region_loading  &&  !is_eu) && <MasterAffilateCard />}
```

Reply:
Same effect with and without loading condition. I think it is to not use loading here. Other components also don't have this.

Reply:
sure, 
few days back i faced a flickering issue , so I just want to double confirm ,
if there is no flickering no need to add